discord_bots/cogs/notification.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+from discord import Colour, Embed, Interaction, app_commands
+from discord.ext.commands import Bot
+from sqlalchemy.orm.session import Session as SQLAlchemySession
+
+from discord_bots.checks import is_command_channel
+from discord_bots.cogs.base import BaseCog
+from discord_bots.models import Queue, QueueNotification, Session
+
+_log = logging.getLogger(__name__)
+
+
+class ListCommands(BaseCog):
```

Comment:
This should be `class NotificationCommands`

Reply:
Good pickup. I was copying the class structure from file to file and must have missed the rename on this one, thanks!
Weird that it didn't give any kind of error though, I guess cause on sync it's syncing the command tree via the unique group.